src/de.rs

```diff
@@ -741,6 +748,7 @@ struct EnumAccess<'de, 'document, 'variant> {
     de: &'variant mut DeserializerFromEvents<'de, 'document>,
     name: Option<&'static str>,
     tag: &'document str,
+    has_visited: bool, // This is a hack to run visitor.visit_map() instead of visitor.visit_enum() when is_serde_content is true
```

Comment:
Why not doc comment? rust-analyzer then can pick-up it and show in intellisence hints

Reply:
Okay. I have updated it.